verilog/parser/verilog_token.h

```diff
@@ -0,0 +1,29 @@
+// Copyright 2017-2020 The Verible Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef VERIBLE_VERILOG_TOKEN_VERILOG_TOKEN_H__
+#define VERIBLE_VERILOG_TOKEN_VERILOG_TOKEN_H__
+
+#include <string>
+
+#include "verilog/parser/verilog_token_enum.h"
+
+namespace verilog {
+
+// Stringifier for verilog_tokentype
+std::string TokenTypeToString(verilog_tokentype tokentype);
```

Comment:
`verilog_symbol_name` returns strings suitable for error messages, which look a bit out of place when used as token type ID in JSON structure (e.g. ["// end of line comment"](https://github.com/google/verible/blob/1e17daab29d1cab50fe017bb146e96d0b90a55ce/verilog/parser/verilog.y#L699)).
In the meantime, I've changed `TokenTypeToString` implementation. It returns token name (e.g. `TK_EOL_COMMENT`) for those few tokens with unusual aliases, and a string produced by `verilog_symbol_name` with stripped quotes (if present) for the rest.

I wonder if it would be OK to remove aliases for comment, newline, and space tokens from [`verilog.y`](https://github.com/google/verible/blob/1e17daab29d1cab50fe017bb146e96d0b90a55ce/verilog/parser/verilog.y#L699) and use `verilog_symbol_name` directly (with quote removing code moved to JSON generating code)?

Reply:
I think in spirit, it would be good to distinguish symbol strings with and without those alternatives, and have a version that only prints the enum names.  Let's follow up after this PR and leave things as-is for now.